### PR TITLE
chore: swap name of 'e2e'  and 'integration' tests

### PR DESCRIPTION
**Pull request by sainoe**
## Linked issues

Closes: #359

## Type of change

- [x] `Refactor`: Changes existing code style, naming, structure, etc.

## Requirements
- [x] change the names
- [ ] inform users of the testing framework

### CODE REVIEW

**Comment by sainoe** [comment]
Note these changes will break the ICS import paths in the [Gaia integration tests](https://github.com/cosmos/gaia/blob/f7f64c1e8b8ed4026a82ef62538ad010f3bc0458/tests/ics/interchain_security_test.go#L8).

File: app/consumer-democracy/app.go [hunk 1]
```diff
@@ -758,50 +759,50 @@ func (app *App) SimulationManager() *module.SimulationManager {
...
 
 // GetConsumerKeeper implements the ConsumerApp interface.
 func (app *App) GetConsumerKeeper() consumerkeeper.Keeper {
 	return app.ConsumerKeeper
 }
 
-// GetE2eBankKeeper implements the ConsumerApp interface.
-func (app *App) GetE2eBankKeeper() e2e.E2eBankKeeper {
+// GetIntgBankKeeper implements the ConsumerApp interface.
+func (app *App) GetIntgBankKeeper() intgutil.IntgBankKeeper {
```
Comment (glnro) [inline app/consumer-democracy/app.go:770]: This is trivial but I'd probably just call this `TestBankKeeper` (same prefix for other test keepers). E2E tests are treating Gaia like a black box anyway.

File: app/consumer/app.go [hunk 2]
```diff
@@ -78,13 +78,13 @@ import (
...
 	dbm "github.com/tendermint/tm-db"
 
-	"github.com/cosmos/interchain-security/testutil/e2e"
 	ibcconsumer "github.com/cosmos/interchain-security/x/ccv/consumer"
 	ibcconsumerkeeper "github.com/cosmos/interchain-security/x/ccv/consumer/keeper"
 	ibcconsumertypes "github.com/cosmos/interchain-security/x/ccv/consumer/types"
 
 	// unnamed import of statik for swagger UI support
 	_ "github.com/cosmos/cosmos-sdk/client/docs/statik"
+	intgutil "github.com/cosmos/interchain-security/testutil/integration"
```
Comment (glnro) [inline app/consumer/app.go:87]:
```suggestion
	testutil "github.com/cosmos/interchain-security/testutil/integration"
```

**Review by glnro — APPROVED** [review]
lgtm, just left a semantic opinion but feel free to disregard

**Comment by glnro** [comment]
> Note these changes will break the ICS import paths in the [Gaia integration tests](https://github.com/cosmos/gaia/blob/f7f64c1e8b8ed4026a82ef62538ad010f3bc0458/tests/ics/interchain_security_test.go#L8).

Will have to coordinate a pr changing this...we can queue something already and point to the last commit on this branch.

File: docs/testing.md [hunk 3]
```diff
@@ -8,64 +8,64 @@ Unit tests are useful for simple standalone functionality, and CRUD operations.
 
 [Mocked external keepers](./testutil/keeper/mocks.go) (implemented with [gomock](https://github.com/golang/mock)) are available for testing code that briefly interacts with external modules, but still only a single function/method relevant to ccv, and a single chain. Ie. do not use mocked external keepers to test the integration of the ccv module with external modules, or integration between consumer and provider.
 
-## End to End (e2e) Tests
+## Integration Tests
```
Comment (smarshall-spitzbart) [inline docs/testing.md:11]: When this PR is up to date with main, the link to this file will be broken in the top level readme. Might be worth fixing in this PR

**Review by smarshall-spitzbart — APPROVED** [review]
Great work, all seems safe to me and it looks like you updated the appropriate documentation

File: app/consumer-democracy/app.go (hunk 1 above)
Comment (sainoe) [reply app/consumer-democracy/app.go:770]: Good point

**Pull request merged**